src/global/tracking/TrackPropagation_factory.cc

```diff
@@ -57,13 +57,14 @@ void eicrecon::TrackPropagation_factory::Process(const std::shared_ptr<const JEv
 
     for(auto traj: trajectories) {
         edm4eic::MutableTrackSegment this_propagated_track;
-        for(size_t isurf = 0; auto surf: m_target_surface_list) {
+        for(auto& surf : m_target_surface_list) {
             auto prop_point = m_track_propagation_algo.propagate(traj, surf);
             if(!prop_point) continue;
-            prop_point->surface = m_target_surface_ID[isurf];
-            prop_point->system = m_target_detector_ID[isurf];
+#if EDM4EIC_VERSION_MAJOR >= 3
```

Comment:
Didn't we remove this already in main? @veprbl

Reply:
Yes. And the constant would be undefined because the include is gone too.